data/en/directoryrename.json

```diff
@@ -1,16 +1,17 @@
 {
 	"name":"directoryRename",
 	"type":"function",
-	"syntax":"directoryRename(path, newPath)",
+	"syntax":"directoryRename(oldPath, newPath)",
 	"returns":"void",
 	"related":["fileMove","directoryCreate","directoryDelete"],
-	"description":" Renames given directory",
+	"description":"Renames given directory. \rNOTE:Ensure that you have the required permissions to run this function.",
 	"params": [
-		{"name":"path","description":"","required":true,"default":"","type":"string","values":[]},
-		{"name":"newPath","description":"","required":true,"default":"","type":"string","values":[]}
+		{"name":"oldPath","description":"Absolute path of the directory to be renamed. Alternatively, you can specify IP address, for example, `DirectoryRename(\"//12.3.123.123/c_drive/test\");`","required":true,"default":"","type":"string","values":[]},
+		{"name":"newPath","description":"New name for the directory.","required":true,"default":"","type":"string","values":[]},
+		{"name":"createPath","description":"Lucee Only. If set to `false`, expects all parent directories to exist. `true` will generate necessary directories.","required":false,"default":"true","type":"boolean","values":[]}
 	],
 	"engines": {
-		"coldfusion": {"minimum_version":"9", "notes":"", "docs":"https://helpx.adobe.com/coldfusion/cfml-reference/coldfusion-functions/functions-c-d/DirectoryRename.html"},
+		"coldfusion": {"minimum_version":"9", "notes":"CF2018 renamed `path` to `oldPath` and `newName` to `newPath`.", "docs":"https://helpx.adobe.com/coldfusion/cfml-reference/coldfusion-functions/functions-c-d/DirectoryRename.html"},
```

Comment:
Named parameters were introduced in CF2018 so I'm not sure what it means to have the name changed.

Reply:
Changes made.  CF2018 comment removed.